src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -310,6 +312,101 @@ public AuthVehicleInfo build() {
     }
   }
 
+  /**
+   * A class that creates a custom AuthUrlConfig object, which is used
+   * when generating an authentication URL.
+   */
+  public static class AuthUrlConfig {
+    private boolean forcePrompt;
+    private String make;
+    private boolean singleSelect;
+    private String state;
+    
+    /** 
+     * A function that creates a map with query params and their values.
+     */
+    public HashMap getParams() {
+      HashMap<String, String> list = new HashMap<>();
```

Comment:
Not sure if this should be called `list`

Reply:
renamed it to `configOptions`